<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DifficultyChoiseMenuWidget</class>
 <widget class="QWidget" name="DifficultyChoiseMenuWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>850</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>20</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #000000
</string>
  </property>
  <widget class="QPushButton" name="EasyButton">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>200</y>
     <width>280</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: url(:/Images/Images/DiffEasy.png); border: 0;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="MediumButton">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>308</y>
     <width>280</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: url(:/Images/Images/DiffMedium.png); border: 0;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="ImpossibleButton">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>421</y>
     <width>280</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: url(:/Images/Images/DiffImpossible.png); border: 0;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="Description">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>190</y>
     <width>256</width>
     <height>300</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #666661; border-style: #000000;</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
